packages/devextreme/js/__internal/grids/grid_core/editing/m_editing_cell_based.ts

```diff
@@ -144,8 +147,9 @@ const editingControllerExtender = (Base: ModuleType<EditingController>) => class
   }
 
   protected _closeEditItem($targetElement) {
+    const isFilterCell = !!$targetElement.closest(`.${this.addWidgetPrefix(FILTER_ROW_CLASS)}`).length;
```

Comment:
Let's move the variable declaration under the condition:
```
if (this._needToCloseEditableCell($targetElement)) {
    const isFilterCell = !!$targetElement.closest(`.${this.addWidgetPrefix(FILTER_ROW_CLASS)}`).length;
    ....
```
It will allow not to perform operations which results could not be used in the future